server/src/tests/url.test.js

```diff
@@ -0,0 +1,186 @@
+const { StatusCodes } = require("http-status-codes");
+const request = require("supertest");
+const { expect } = require("chai");
+
+const app = require("../../app");
+const {
+  deleteAllUrls,
+  addNewUrl,
+  findOneByShorterUrl,
+} = require("../services/urls");
+const { REAL_URL, MOCKED_URL, MODIFIED_URL } = require("../utils/consts");
+
+before(async () => {
+  await deleteAllUrls();
+});
+
+beforeEach(async () => {
+  await deleteAllUrls();
+  await addNewUrl(REAL_URL.originalUrl, REAL_URL.shorterUrl);
+});
+
+after(async () => {
+  await deleteAllUrls();
+});
+
+describe("Get", () => {
```

Comment:
we can use just 1 before and not, delete and insert to table for every single GET test. in the end, for GET test, the data remain the same after every test.